modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/SaveContentFragmentRenderer.java

```diff
@@ -63,8 +70,43 @@ public void render(
 		FragmentRendererContext fragmentRendererContext,
 		HttpServletRequest httpServletRequest,
 		HttpServletResponse httpServletResponse) {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-197909")) {
+			return;
+		}
+
+		SavedContentTag savedContentTag = new SavedContentTag();
+
+		savedContentTag.isViewMode(fragmentRendererContext.isViewMode());
+
+		Tuple displayObject = getDisplayObject(
+			fragmentRendererContext, httpServletRequest);
+
+		savedContentTag.setClassName(
+			GetterUtil.getString(displayObject.getObject(0)));
+
```

Comment:
Optional: I think this newline shouldn't be here; both set* methods take its input from the same place.

Reply:
thanks, I will add an commit removing this newline